src/types.ts

```diff
@@ -1874,6 +1874,26 @@ export interface TokenLink {
     displayName?: string;
 }
 
+export enum CollectionType {
+    NON_FUNGIBLE_TOKEN = "NON_FUNGIBLE_TOKEN",
+    SEMI_FUNGIBLE_TOKEN = "SEMI_FUNGIBLE_TOKEN",
+}
+
+export interface CollectionLink {
+    id: string;
+    type: CollectionType;
+    refId: string;
```

Comment:
I believe we decided not to have refId in the CollectionLInk to avoid confusion with multiple Ids.

Reply:
@or109  Not sure if you got a chance to check this comment.